src/app/page.tsx

```diff
@@ -32,12 +32,6 @@ export default function Home() {
 		setIsModalOpen(true);
 	};
 
-	/**
-	 * TODO: Need to somehow know if contract has been deployed to chain.
```

Comment:
is this not needed anymore? or was this implemented elsewhere?

Reply:
Still need to do this, but I have made a note on how to do it elsewhere so that this comment can be removed.